src/test/java/tests/AddPayGradeTest.java

```diff
@@ -0,0 +1,23 @@
+package tests;
+
+import library.TestBase;
+import org.testng.annotations.Test;
+import pages.HeaderPage;
+import pages.LoginPage;
+import pages.MenuOptions;
+import pages.PayGradePage;
+
+public class AddPayGradeTest extends TestBase {
+    @Test
+    public void addPayGrade(){
+        LoginPage loginPage = new LoginPage(driver);
+        loginPage.login("Admin","admin123", true, null);
+        PayGradePage payGradePage = new PayGradePage(driver);
+        payGradePage.payGradePageNav();
+        payGradePage.addNewPayGrade("Grade 6");
+
```

Comment:
Assertions are missing after the tests
kindly see the customers automation done in this batch , and accordingly include all feedbacks